README.md

```diff
@@ -102,6 +102,21 @@ Sample setting:
 ```
 "checker-framework.languageserver_repo": "checker-framework-languageserver"
 ```
+## Developer's Guide
+
+During development, you can simply start the program from the command line and run as a new instance:
+1.  Open up this project (checker-framework-vscode) on VSCode and start a new instance by clicking on `Run` in the top bar menu, then `Start Debugging`. 
+2.  Open a new folder with a `.java` (this can be a simple Hello World folder) file in the VSCode with `[Extension Development Host]` instance of VSCode.
+
+The manual method involves the following steps:
+To add from the command line, you can install the extension on VSCode:
+1. Install its dependencies: `yarn install`
+2. Run the following script `yarn global add vsce`
```

Comment:
This simply installs the CLI tool but it's not going to install the extension.

According to https://code.visualstudio.com/api/working-with-extensions/publishing-extension#packaging-extensions, you'll still need to package the extension with `vsce package` and install it via `code --install-extension my-extension-0.0.1.vsix`